packages/gen-ai/frontend/.eslintrc.js

```diff
@@ -230,6 +230,13 @@ module.exports = {
     'func-names': 'warn',
   },
   overrides: [
+    {
```

Comment:
You probably don't need this change after the fix in this PR - https://github.com/opendatahub-io/odh-dashboard/pull/4830.

Reply:
`npm test` failing locally